README.md

```diff
@@ -115,8 +115,8 @@ I'm also going to be pretty conservative about what I add.
 
 This plugin must be explicitly enabled by using `require("toggleterm").setup{}`
 
-Setting the `open_mapping` key to use for toggling the terminal(s) will set up mappings for _normal_ mode
-If you prefix the mapping with a number that particular terminal will be opened.
+Setting the `open_mapping` key to use for toggling the terminal(s) will set up mappings for _normal_ mode.
+If you prefix the mapping with a number that particular terminal will be opened. Othrwise if a prefix is not set, then the last toggled terminal will be opened.
```

Comment:
```suggestion
If you prefix the mapping with a number that particular terminal will be opened. Otherwise if a prefix is not set, then the last toggled terminal will be opened.
```